logdetective/server/database/models/merge_requests.py

```diff
@@ -0,0 +1,462 @@
+import enum
+import datetime
+from typing import Optional
+from sqlalchemy import (
+    Enum,
+    Column,
+    Integer,
+    DateTime,
+    String,
+    ForeignKey,
+    UniqueConstraint,
+    desc,
+)
+from sqlalchemy.orm import relationship
+from logdetective.server.database.base import Base, transaction
+
+
+class Forge(str, enum.Enum):
+    """List of forges managed by logdetective"""
+
+    gitlab_com = "https://gitlab.com"  # pylint: disable=(invalid-name)
+    gitlab_cee_redhat_com = "http://gitlab.cee.redhat.com/"  # pylint: disable=(invalid-name)
+
+
+class GitlabMergeRequests(Base):
```

Comment:
We might want to call this `GitlabMergeRequestJobs` instead, since the `(forge, project, mr)` combination may appear multiple times, representing multiple job runs of the same merge request.